### PR TITLE
Fix params passed to plugins when yum proxy is set

### PR DESCRIPTION
**Pull request by vrutkovs**

### CODE REVIEW

**Comment by twaugh** [comment]
Good catch. The unit test is also incorrect.

**Comment by vrutkovs** [comment]
Right, updated the unittests

File: osbs/build/build_request.py [hunk 1]
```diff
@@ -563,7 +563,7 @@ def render_add_filesystem(self):
             self.dj.dock_json_set_arg(phase, plugin, 'koji_hub',
                                       self.spec.kojihub.value)
             if self.spec.proxy.value:
-                self.dj.dock_json_set_arg(phase, plugin, 'koji_proxyuser',
+                self.dj.dock_json_set_arg(phase, plugin, 'proxy',
```
Comment (lcarva) [inline osbs/build/build_request.py]: `proxy` is not a valid param for `add_filesystem` plugin.

**Comment by vrutkovs** [comment]
Right, removed passing this param entirely. `koji_proxyuser` doesn't seem to used anywhere

**Comment by lcarva** [comment]
Not adding to release notes as this seems to be obscure enough and not impact any users.
Please add it to it if you think otherwise.

**Pull request merged**